src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend_utils.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# flake8: noqa
+# mypy: ignore-errors
+
+from typing import Optional, Any
+import os
+from openvino.runtime import Core
+
+
+def _get_device(options) -> Optional[Any]:
+    core = Core()
+    device = "CPU"
+
+    if "device_id" in options:
+        device = options["device_id"]
```

Comment:
Pytorch usually call this `device` I think it will be better to align with pytorch conventions instead of `device_id`

Reply:
Made the change